native/packages/react-native-bpk-component-card/src/BpkCard.js

```diff
@@ -72,7 +73,10 @@ const BpkCard = (props) => {
   }
 
   return (
-    <View
+
+    <TouchableHighlight
```

Comment:
If this is pressable should it have a "button" role for accessibility? cc @shaundon

Reply:
It should yeah - @matteo-hertel happy to give you a hand if you like

Reply:
@shaundon yes, please!